lib/ansible/plugins/connection/psrp.py

```diff
@@ -119,7 +119,7 @@
       (First sleeps X, then sleeps 2*X, then sleeps 4*X, ...)
     - This is measured in seconds.
     vars:
-    - name: ansible_psrp_connection_backoff
+    - name: ansible_psrp_reconnection_backoff
```

Comment:
You can't change it now that it's been in a release, you can only just add a new variable and keep the existing one.

Reply:
Trying to fix the issue #58714 - seems he's right and the module does refer to a different var

Reply:
The options doc section actually define the vars that are used to control that option. The name of the var can be anything and typically we do just name like `ansible_<connection>_<option>` but that didn't happen in this case.

This means that you can still set `ansible_psrp_connection_backoff` to control this option and this PR makes it so you cannot. We need to ensure that it is backwards compatible which means keeping the older var and just adding a new one called `ansible_psrp_reconnection_backoff`.

Reply:
I see. I'm gonna close this PR and see what develops in the originating issue